data/human/hails.txt

```diff
@@ -1670,6 +1670,47 @@ phrase "friendly civilian"
 		"The next event will be great!"
 		"That's real entertainment!"
 
+phrase "friendly civilian"
+	word
+		"I"
+		"Me and my friend"
+	word
+		" "
+	word
```

Comment:
There is a mix of jobs and hobbies/personal projects here, and using them interchangeably in a phrase feels off-putting to me. A couple examples:
`worked in the business of cleaning corridors in an university`
`used to spend time on farming in the dirt belt`
There's a good variety here, but the downside is that some of these possible combinations sound really odd. You could remove some of the "words" at this part (I think simply removing `used to spend time on` would work), entirely change this up to make separate phrases for hobby-based things and work-based things, or remove the the "words" later that specifically reference jobs rather than things that could be jobs or hobbies (like farming, making ketchup, etc.).

Reply:
Good feedback! I removed "used to spend time" and I removed "worked in the business of".